### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by zheniasemenina**
Added a note on Start/End Session events to clarify that they do count towards the event volume.

# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

**Comment by markzegarelli** [comment]
@zheniasemenina I think we need to work on the whole paragraph that comes before your addition, as it's pretty confusing to me.

It starts by saying Amplitude sends events automatically, but in the next sentence says "even though these events aren't sent..."

Is it correct to say that Amplitude tracks `[Amplitude] Session Start` and `[Amplitude] Session End` by default, and these events count toward the allotted volume? If so, where does `session_id` come into play in this context?

Same issue in #114

**Comment by zheniasemenina** [comment]
@markzegarelli 
> Is it correct to say that Amplitude tracks `[Amplitude] Session Start` and `[Amplitude] Session End` by default, and these events count toward the allotted volume? If so, where does `session_id` come into play in this context?

Right, we track `[Amplitude] Session Start` and `[Amplitude] Session End` by default, and these events count toward the allotted volume. 
We use `session_id` to distinguish between different sessions (all events with the same session_id are grouped together into the same session), and it comes into play because `[Amplitude] Session Start` generates a unique `session_id`  every time a new session starts (and `[Amplitude] Session End` means this session is over and it will be the last event with that same session number)

**Comment by markzegarelli** [comment]
@zheniasemenina Cool! I think the context around this paragraph is mainly around how you can disable those defaults. 

How about something like this:

> By default, Amplitude sends `[Amplitude] Start Session` and `[Amplitude] End Session` events. These events count toward your contracted event volume. When you disable default session tracking, Amplitude no longer sends session start and end events, but continues to add `session_id` as an event property.

Is that correct regarding `session_id`?

**Comment by zheniasemenina** [comment]
@markzegarelli  that's right!

**Review by markzegarelli — APPROVED** [review]

File: docs/data/sdks/ios-swift/index.md [hunk 1]
```diff
@@ -704,8 +704,7 @@ You can adjust the time window for which sessions are extended. The default sess
     Amplitude* amplitude = [Amplitude initWithConfiguration:configuration];
     ```
 
-By default, Amplitude automatically sends the '[Amplitude] Start Session' and '[Amplitude] End Session' events. Even though these events aren't sent, sessions are still tracked by using `session_id`.
-You can also disable those session events.
+By default, Amplitude sends `[Amplitude] Start Session` and `[Amplitude] End Session` events. These events count toward your contracted event volume. When you disable default session tracking, Amplitude no longer sends session start and end events, but continues to add `session_id` as an event property.
```
Comment (markzegarelli) [inline docs/data/sdks/ios-swift/index.md:707]:
```suggestion
By default, Amplitude sends `[Amplitude] Start Session` and `[Amplitude] End Session` events. These events count toward your contracted event volume. When you disable default session tracking, Amplitude no longer sends session start and end events, but continues to add session_id as an event property.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

File: docs/data/sdks/ios-swift/index.md [hunk 2]
```diff
@@ -704,8 +704,7 @@ You can adjust the time window for which sessions are extended. The default sess
     Amplitude* amplitude = [Amplitude initWithConfiguration:configuration];
     ```
 
-By default, Amplitude automatically sends the '[Amplitude] Start Session' and '[Amplitude] End Session' events. Even though these events aren't sent, sessions are still tracked by using `session_id`.
-You can also disable those session events.
+By default, Amplitude sends `[Amplitude] Start Session` and `[Amplitude] End Session` events. These events count toward your contracted event volume. When you disable default session tracking, Amplitude no longer sends session start and end events, but continues to add session_id as an event property.
```
Comment (markzegarelli) [inline docs/data/sdks/ios-swift/index.md:707]:
```suggestion
By default, Amplitude sends `[Amplitude] Start Session` and `[Amplitude] End Session` events. These events count toward your contracted event volume. When you disable default session tracking, Amplitude no longer sends session start and end events, but continues to add `session_id` as an event property.
```

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**